js/services/contact_service.js

```diff
@@ -93,6 +93,9 @@ angular.module('contactsApp')
 		newContact.uid(newUid);
 		newContact.setUrl(addressBook, newUid);
 		newContact.addressBookId = addressBook.displayName;
+		if (_.isUndefined(newContact.fullName()) || newContact.fullName() === '') {
+			newContact.fullName(t('contacts', 'New Contact'));
```

Comment:
This should be capitalized like »New contact« – we always only capitalize the first word. ;)